.github/workflows/ci.yml

```diff
@@ -19,6 +19,36 @@ jobs:
     - name: module api test
       run: ./runtest-moduleapi --verbose
 
+  test-sanitizer-address:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: make
+        run: make SANITIZER=address REDIS_CFLAGS='-DREDIS_TEST'
+      - name: testprep
+        run: sudo apt-get install tcl8.6 tclx -y
+      - name: test
+        run: ./runtest --sanitizer --verbose --tags -slow
+      - name: module api test
+        run: ./runtest-moduleapi --sanitizer --verbose
+      - name: unittest
+        run: ./src/redis-server test all
+
+  test-sanitizer-undefined:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: make
+        run: make SANITIZER=undefined REDIS_CFLAGS='-DREDIS_TEST'
+      - name: testprep
+        run: sudo apt-get install tcl8.6 tclx -y
+      - name: test
+        run: ./runtest --sanitizer --verbose --tags -slow
+      - name: module api test
+        run: ./runtest-moduleapi --sanitizer --verbose
+      - name: unittest
+        run: ./src/redis-server test all
```

Comment:
these are kinda rare, i think we can keep relying on the daily CI for them.

Reply:
@oranagra Just to be clear, do you want `./runtest-moduleapi` here or not?

Reply:
sorry i wasn't clear.
i meant that i don't think the undefined sanitizer should be executed in the PR CI.
I wanted the address sanitizer since it can find memory leaks before they cause failures in the daily CI.
but i think the UB sanitizer issues are rarer and we can skip that and rely on the daily CI.

regarding module tests, (in the address sanitizer) i do want them. and ideally i would have wanted cluster testss too, but they're just too slow.

Reply:
Okay. Removed ubsan from the CI.

Just fyi, `runtest --tags -slow` takes ~7 minutes. Cluster tests are ~13 minutes and sentinel tests are ~2 minutes : https://github.com/tezc/redis/runs/4079444886. 

Let me know if you want to add these.

Reply:
i see [here](https://github.com/redis/redis/runs/4094541432?check_suite_focus=true) that the runtest part takes 5 minutes (with and without sanitizer takes the same time and they run in parallel).
if we'll add runtest-cluster and runtest-sentinel, we'll add them on the same build (sequentially), so the CI would take 20 minutes instead of 5.

the problem with these cluster and sentinel tests is that they run each of the units sequentially rather than in parallel to each one.. one day someone will have the time to fix that, but for now, i rather leave the CI fast (5 minutes)

Reply:
gotcha